scripts/generate_json_api.py

```diff
@@ -128,45 +230,104 @@ class Examples(CommandTestCase):
         await self.on_transaction_dict(stream)
 
         await r(
-            'List all your claims.',
+            'List all your claims',
             'claim', 'list'
         )
 
         await r(
-            'Paginate your claims.',
+            'Paginate your claims',
             'claim', 'list', '--page=1', '--page_size=20'
         )
 
         await r(
-            'List all your stream claims.',
+            'List all your stream claims',
             'stream', 'list'
         )
 
         await r(
-            'Paginate your stream claims.',
+            'Paginate your stream claims',
             'stream', 'list', '--page=1', '--page_size=20'
         )
 
         await r(
-            'List all your channel claims.',
-            'channel', 'list'
+            'Search for all claims in channel',
+            'claim', 'search', f"--channel_id=\"{channel_id}\""
         )
 
         await r(
-            'Paginate your channel claims.',
-            'channel', 'list', '--page=1', '--page_size=20'
+            'Search for claims matching a name',
+            'claim', 'search', f"--name=\"{stream_name}\""
         )
 
+
+        # files
+
+        file_list_result = await r(
+            'List local files',
+            'file', 'list'
+        )
+        file_uri = f"{file_list_result[0]['claim_name']}#{file_list_result[0]['claim_id']}"
+        await r(
+            'Resolve a claim',
+            'resolve', file_uri
+        )
+
+        await r(
+            'List files matching a parameter',
+            'file', 'list', f"--claim_id=\"{file_list_result[0]['claim_id']}\""
+        )
+
+        await r(
+            'Delete a file',
+            'file', 'delete', f"--claim_id=\"{file_list_result[0]['claim_id']}\""
+        )
+
+        await r(
+            'Get a file',
+            'get', file_uri
+        )
+
+        # blobs
+
+        bloblist = await r(
+            'List Blobs',
+            'blob', 'list'
+        )
+        # NOT IMPLEMENTED
+        # await r(
+        #     'Reflect a blob',
+        #     'blob', 'reflect', f"{bloblist}"
+        # )
+
+        # NO DHT
+        # await r(
+        #     'Get a blob',
+        #     'blob', 'get', f"\"{bloblist[0]}\""
+        # )
+        # await r(
+        #     'Announce a blob',
+        #     'blob', 'announce', f"--blob_hash={bloblist[0]}"
+        # )
+        await r(
+            'Delete a blob',
+            'blob', 'delete', f"{bloblist[0]}"
+        )
+
+
+        # support
+
+        # destroy
```

Comment:
what's destroy?

Reply:
# destroy was going to be things like abandon or delete type things. 

I wanted to organize the script into meaningful categories, but maybe that doesn't make sense.